### ISSUE TITLE
Use tmux instead of GNU screen by default in ttys?

### ISSUE THREAD

**Issue by mika** [grml/grml#157]
We are starting `GNU screen` sessions on some TTYs after booting finished.
I'm fine with `GNU screen`, though `tmux` is a more modern tool which also provides some features `screen` doesn't provide.
Assuming that we manage to get tmux as proper screen replacement (similar status line + keybindings, comparable memory usage,...), do we want to switch to use tmux on our TTYs? Opinions? /cc @jkirk @formorer @zeha?

**Comment by zeha** [comment]
👍🏻 

Q because I have no idea if that works / is needed: tmux on serial consoles?

**Comment by mika** [comment]
Good point, `tmux` doesn't support serial consoles like `screen` does, though AFAICT you'd invoke it separately anyway (like `screen /dev/ttyUSB0 115200`), so IIRC it should *not* matter for the ttys where we're invoking it. Please correct me if I'm wrong. :)

**Comment by zeha** [comment]
I was worried about the other way round: does tummy work on a serial console? Do we care?

**Comment by mika** [comment]
Ahhhh! :) AFAICS we only have `agetty` listening on `serial-getty@.service`, so that shouldn't be relevant.

**Comment by jkirk** [comment]
JFTR, tmux does (and will not) support serial console (https://github.com/tmux/tmux/issues/1862).

The question is, do we need tmux to work on a serial console?

But yes, I also think we should switch to tmux.

**Comment by zeha** [comment]
> The question is, do we need tmux to work on a serial console?

Per mika above, this is not relevant.
